ts/components/ui/Markdown/index.tsx

```diff
@@ -298,9 +298,7 @@ class Markdown extends React.PureComponent<Props, State> {
     const containerStyle: ViewStyle = {
       height: htmlBodyHeight + (extraBodyHeight || 0)
     };
-
-    const isLoading =
-      html === undefined || (html !== "" && htmlBodyHeight === 0);
+    const isLoading = html === undefined;
```

Comment:
I believe we should keep the condition `htmlBodyHeight === 0` to determine if it is still in loading
the html body height is computed by injecting some JS into the webview, listen for incoming results and then apply those results to the component height

here a comparison

### without that condition (the showing is less smooth and you can see some frames with a white background)
https://user-images.githubusercontent.com/822471/142253694-c1714917-63c4-4371-b698-382cf1f95170.mov

### with that condition
https://user-images.githubusercontent.com/822471/142253731-f58339f4-f510-4523-b7f6-acfd93cf0382.mov

Reply:
The problem is that if we keep the condition `htmlBodyHeight === 0` we return to the infinite loading because the `htmlBodyHeight` is always `0` when the markdown text contains only spaces.
I've tried another approach in https://github.com/pagopa/io-app/pull/3502/commits/cf0ef03e2cb203b780a9110a5e980355a89f42c1 and https://github.com/pagopa/io-app/pull/3502/commits/8c2548f0cd915d3aa2ff7c4ec81788df97e350ff, using the `handleWebViewMessage` to choose when the loading is completed that seems to have the same graphical behaviour:

https://user-images.githubusercontent.com/26501317/142438004-5e26573b-2191-4c2b-b6bc-5108f2a75da4.mov